tests/integration/cmd_debug_test.go

```diff
@@ -60,9 +59,10 @@ var _ = Describe("odo debug command tests", func() {
 
 		It("should expect a ws connection when tried to connect on different debug port locally and remotely", func() {
 			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
-			helper.CmdShouldPass("odo", "component", "create", "--s2i", "nodejs:latest", "--project", commonVar.Project, "--context", commonVar.Context)
-			helper.CmdShouldPass("odo", "config", "set", "--force", "DebugPort", "9292", "--context", commonVar.Context)
-			dbgPort := helper.GetConfigValueWithContext("DebugPort", commonVar.Context)
+			helper.CmdShouldPass("odo", "component", "create", "--s2i", "nodejs", "--project", commonVar.Project, "--context", commonVar.Context)
+			helper.CmdShouldPass("odo", "env", "set", "--force", "DebugPort", "9292", "--context", commonVar.Context)
```

Comment:
why we are setting debug port twice?

Reply:
because the `env set DebugPort 9292` doesn't work. I am debugging that and would file a bug report if confirmed

Reply:
opened an issue https://github.com/openshift/odo/issues/4615, confirming the setting of that value isn't working